components/layout_2020/primitives.rs

```diff
@@ -1,28 +1,29 @@
-use crate::text;
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at https://mozilla.org/MPL/2.0/. */
 
-/// Origin at top-left corner, unit `1px`
-pub struct CssPx;
+// use crate::text;
 
-pub use euclid::point2 as point;
-pub use euclid::rect;
-pub type Length<U> = euclid::Length<f32, U>;
-pub type Point<U> = euclid::TypedPoint2D<f32, U>;
-pub type Size<U> = euclid::TypedSize2D<f32, U>;
-pub type Rect<U> = euclid::TypedRect<f32, U>;
-pub type SideOffsets<U> = euclid::TypedSideOffsets2D<f32, U>;
-pub type Scale<Src, Dest> = euclid::TypedScale<f32, Src, Dest>;
+// pub use euclid::point2 as point;
+// pub use euclid::rect;
+// pub type Length<U> = euclid::Length<f32, U>;
+pub type Point<U> = euclid::Point2D<f32, U>;
+pub type Size<U> = euclid::Size2D<f32, U>;
+pub type Rect<U> = euclid::Rect<f32, U>;
+// pub type SideOffsets<U> = euclid::SideOffsets2D<f32, U>;
+// pub type Scale<Src, Dest> = euclid::Scale<f32, Src, Dest>;
 
-#[derive(Copy, Clone, PartialEq)]
-pub struct RGBA(pub f32, pub f32, pub f32, pub f32);
+// #[derive(Copy, Clone, PartialEq)]
+// pub struct RGBA(pub f32, pub f32, pub f32, pub f32);
```

Comment:
Let’s remove the parts of this file that are commented out (we’re unlikely to use them as-is) and move the rest to `geom.rs`.

`primitives.rs` was first created for Victor’s PDF backend.